docs/docs/prior-art-and-alternatives.md

```diff
@@ -7,25 +7,25 @@ sidebar_label: Prior Art / Alternatives
-| <!-- 2018-07-05 --> <a rel="nofollow" href="https://github.com/tibdex/autorebase">Autorebase</a>                                                   | ✅           | ✅                   | ❌                          | ✅                           | ❌               | ❌                   | ✅          | ❌          | ❌                                        | TypeScript |
-| <!-- 2018-09-21 --> <a rel="nofollow" href="https://github.com/SvanBoxel/auto-merge">Auto Merge</a>                                                | ✅           | ❌                   | ❌                          | ❌                           | ❌               | ❌                   | ✅          | ❌          | ❌                                        | JavaScript |
-| <!-- 2018-10-21 --> <a rel="nofollow" href="https://github.com/phstc/probot-merge-when-green">Merge when green</a>                                 | ✅           | ❌                   | ❌                          | ❌                           | ❌               | ❌                   | ✅          | ❌          | ❌                                        | JavaScript |
-| <!-- 2020-12-16 --> <a rel="nofollow" href="https://github.blog/changelog/2020-12-16-pull-request-auto-merge-public-beta/">GitHub's auto-merge</a> | ✅           | ❌                   | ❌                          | ✅                           | ✅               | ✅                   | ❌          | ✅          | ✅                                        | 🤷‍        |
-| <!-- 2021-08-28 --> <a rel="nofollow" href="https://mergequeue.com">MergeQueue</a>                                                                 | ✅           | ✅                   | ✅                          | ✅                           | ❌               | ❌                   | ❌          | ✅          | ❌                                        | 🤷‍        |
-| <!-- Unknown --> <a rel="nofollow" href="https://reporanger.com">Ranger</a> ‍                                                                      | ✅           | ❌ ‍                 | ❌ ‍                        | ✅                           | ❌               | ❌                   | ❌          | ✅          | ❌                                        | 🤷‍        |
+| Name                                                                                                                                               | Auto Merging | Auto Update Branches | Update Branches Efficiently | Works With Branch Protection | Works with Forks | Simple Configuration | Open Source | Hosted SaaS | Works with GitHub | On-prem possible? | Project alive? | Free for public and personal repositories | Language   |
```

Comment:
looks good, thanks for adding the extra columns

one minor thing is I'd prefer the order to be chronological (with the exception of Kodiak) based on their creation data which is in the html comment in the first column

Reply:
Done, reordered from new to old. Also updated the date for MergeQueue as it seemed to be in the future (end of August'21), the first blog article was around January, so I put January'21